tests/manage/z_cluster/nodes/test_nodes_restart_ms.py

```diff
@@ -295,3 +295,30 @@ def test_node_maintenance_restart(self, nodes, pvc_factory, pod_factory):
 
         # Create PVCs and pods
         self.sanity_helpers.create_resources_on_ms_consumers()
+
+    @tier4b
+    @pytest.mark.polarion_id("OCS-2015")
+    def test_rolling_provider_worker_nodes_restart(self, nodes):
```

Comment:
Is this test only for Managed Service cluster or can it be used also on normal cluster?
If it is only for MS cluster, we should probably skip it on normal cluster via `@managed_service_required`.

Reply:
Missed the `@managed_service_required` mark on the class level.

Reply:
The decorator is on all the methods in the class: https://github.com/red-hat-storage/ocs-ci/blob/master/tests/manage/z_cluster/nodes/test_nodes_restart_ms.py#L45.